internal/store/logs.go

```diff
@@ -82,7 +85,26 @@ func (s *Store) Collect(groupID string, newLog *opsmngr.LogCollectionJob) (*opsm
 func (s *Store) DownloadLog(groupID, host, name string, out io.Writer, opts *atlas.DateRangetOptions) error {
```

Comment:
We have dedicated v2 objects to organize those arguments. Can we use that?

Reply:
Sure, I will use `GetHostLogsWithParams` instead.